src/main.cpp

```diff
@@ -1178,14 +1178,6 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
     if (tx.IsCoinStake())
         return state.DoS(100, false, REJECT_INVALID, "coinstake");
 
-    // Don't relay version 2 transactions until CSV is active, and we can be
```

Comment:
this is the broadcast error i was hitting right? So we are happy to just remove the check? No other impact?